cypress/integration/form-validation/phone-number.spec.js

```diff
@@ -0,0 +1,76 @@
+/// <reference types="cypress" />
+
+
+describe('Form validation', () => {
```

Comment:
These describe blocks, should be they more specific than just 'form validation'? Each example should have its own unique title.